pgmpy/models/BayesianModel.py

```diff
@@ -1030,3 +1030,16 @@ def get_markov_blanket(self, node):
         blanket_nodes = set(blanket_nodes)
         blanket_nodes.remove(node)
         return list(blanket_nodes)
+    
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            network_struct1 = (self.nodes(),self.edges())
```

Comment:
@Vaibhavdixit02 Please follow PEP8 (https://www.python.org/dev/peps/pep-0008/). Needs a space after the comma.